generators/generator-bot-enterprise-assistant/README.md

```diff
@@ -1,68 +1,50 @@
 # @microsoft/generator-bot-enterprise-assistant [![NPM version][npm-image]][npm-url]
```

Comment:
Missing the npm-image/npm-url references, this renders like: 
![image](https://user-images.githubusercontent.com/43043272/117085385-72a4b680-acfe-11eb-82ab-373c75bd409a.png)

Reply:
fixed.